coordinator/src/services/translation/index.ts

```diff
@@ -104,9 +103,10 @@ function createParametersDisplay(fragmentsToDisplay: Display): string {
   return Buffer.from(Mustache.render(displayTemplate, fragmentsToDisplay)).toString("base64")
 }
 
-function createParameters(base64Payload: string, base64Display: string): fhir.Parameters {
+function createParameters(base64Fragments: string, base64Digest: string, base64Display: string): fhir.Parameters {
   const parameters: Array<fhir.Parameter> = []
-  parameters.push({name: "payload", valueString: base64Payload})
+  parameters.push({name: "fragments", valueString: base64Fragments})
+  parameters.push({name: "digest", valueString: base64Digest})
```

Comment:
The digest is no longer required by the Signing Service, but I've left it in case it's needed by the supplier (in order to build the XML-DSig) or by the Remote Signing Service in future